### PR TITLE
small tweaks?

### PR DESCRIPTION
**Pull request by littlewhitecloud**
LOL Maybe someone like me often write `set_theme("Dark")`, so I think we can add `.lower()` to avoid the check fails.
Also think we can use
```python
cls.set_theme("dark" if cls.get_theme() == "light" else "light")
```
instead of
```python
if cls.get_theme() == "light":
    cls.set_theme("dark")
else:
    cls.set_theme("light")
```

### CODE REVIEW

File: sv_ttk/__init__.py [hunk 1]
```diff
@@ -54,18 +54,15 @@ def get_theme(cls, root: tkinter.Misc | None = None) -> str:
     @classmethod
     def set_theme(cls, theme: str, root: tkinter.Misc | None = None) -> None:
         cls.load_theme(root)
-        if theme not in {"dark", "light"}:
+        if theme.lower() not in {"dark", "light"}: # Maybe someone will write Dark
             raise RuntimeError("not a valid sv_ttk theme name: {}".format(theme))
 
         cls.tcl.call("set_theme", theme)
```
Comment (rdbende) [inline sv_ttk/__init__.py:60]:
```suggestion
        cls.tcl.call("set_theme", theme.lower())
```

You will need to add `.lower()` here as well.

Comment (Akuli) [inline sv_ttk/__init__.py:60]:
```suggestion
        cls.tcl.call("set_theme", theme.lower())
```

File: sv_ttk/__init__.py [hunk 2]
```diff
@@ -54,18 +54,15 @@ def get_theme(cls, root: tkinter.Misc | None = None) -> str:
...
 
     @classmethod
     def toggle_theme(cls, root: tkinter.Misc | None = None) -> None:
         cls.load_theme(root)
-        if cls.get_theme() == "light":
-            cls.set_theme("dark")
-        else:
-            cls.set_theme("light")
+        cls.set_theme("dark" if cls.get_theme() == "light" else "light")
```
Comment (rdbende) [inline sv_ttk/__init__.py:65]: I don't really like this. It's shorter, but if expressions also make the code harder to read.

**Review by rdbende — CHANGES_REQUESTED** [review]

File: sv_ttk/__init__.py (hunk 1 above)
Comment (Akuli) [reply sv_ttk/__init__.py:60]: Lol, we posted the exact same suggestion at the same time :D

Comment (littlewhitecloud) [reply sv_ttk/__init__.py]: LOL, akuli give the same change.

Comment (littlewhitecloud) [reply sv_ttk/__init__.py]: Thanks!

File: sv_ttk/__init__.py (hunk 2 above)
Comment (littlewhitecloud) [reply sv_ttk/__init__.py:65]: I don't think it i harder to read. (Actually, at first I also think it is harder to read but Moosems advise me to use it. Later I can write and read it very well.) And I think it is the right syntax for the toggle function.

Comment (littlewhitecloud) [reply sv_ttk/__init__.py:65]: But, if you still think it is hard to read, I can revert it.

**Review by rdbende — APPROVED** [review]

**Comment by rdbende** [comment]
Thanks!

**Pull request merged**